yourgov/pages/app/licences-and-permits/apply/mobile-food-vendor-permit/success/index.tsx

```diff
@@ -0,0 +1,88 @@
+import { Fragment, ReactElement } from 'react';
+import { useRouter } from 'next/router';
+import { Column, Columns } from '@ag.ds-next/react/columns';
+import { DirectionLink } from '@ag.ds-next/react/direction-link';
+import { H2 } from '@ag.ds-next/react/heading';
+import { PageContent } from '@ag.ds-next/react/content';
+import { PageAlert } from '@ag.ds-next/react/page-alert';
+import { Stack } from '@ag.ds-next/react/stack';
+import { Text } from '@ag.ds-next/react/text';
+import { TextLink } from '@ag.ds-next/react/text-link';
+import { AppLayout } from '../../../../../../components/Layout/AppLayout';
+import { DocumentTitle } from '../../../../../../components/DocumentTitle';
+import { HelpCallout } from '../../../../../../components/HelpCallout';
+import {
+	GlobalFormProvider,
+	useGlobalForm,
+} from '../../../../../../components/FormMobileFoodVendorPermit';
+import type { NextPageWithLayout } from '../../../../../_app';
+import { FormContainer } from '../../../../../../components/FormMobileFoodVendorPermit/FormContainer';
+import { managePermitsPage } from '../../../../../../components/FormMobileFoodVendorPermit/utils';
+
+const Page: NextPageWithLayout = () => {
+	const { query, isReady } = useRouter();
+	const { formState } = useGlobalForm();
+
+	const isSuccess = query?.id === formState?.id && formState?.completed;
+
+	return (
+		<Fragment>
+			<DocumentTitle title="Application submitted" />
```

Comment:
Does this need to include the application ID and/or type so that it's unique across all dynamic pages? Similar for the `h1`?

Reply:
It could, but I don't think it's that important